docs/virtualkeyboard/index.html

```diff
@@ -140,47 +159,89 @@ <h2>The <dfn>VirtualKeyboard</dfn> Interface</h2>
-                </li>
+                  </li>
+                  <li>
+                    [=In parallel=], follow these steps:
+                    <ol>
+                        <li>Wait for the virtual keyboard to be hidden by the system.</li>
+                        <li>Call {{VirtualKeyboard/set the virtual keyboard bounding rect}} with the keyboard's OS reported bounding rectangle (which has all 0 values) and the [=document=]'s <a href="https://drafts.csswg.org/css2/#viewport">viewport</a> rectangle.</li>
+                        <li>[=Fire an event=] named "geometrychange" at [=this=].</li>
+                    </ol>
+                   </li>
                 </ol>
                </dd>
                <p class="note">
-                Platform heuristics may impose additional restrictions on VK {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}}. e.g., on Windows the pointer type has to be touch or pen.
+                Platform heuristics may impose additional restrictions on VK {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}}. e.g., on Windows the pointer type MUST be touch or pen.
               </p>
               <p class="note">
                 A few people have <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/415">expressed concerns</a> around {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}} methods not being promise-based, but we believe it is a better design for web developers to use {{VirtualKeyboard/ongeometrychange}} event since it is fired when VK visibility changes, making the return values unnecessary.
               </p>
             <dt>
-              <dfn>overlaysContent</dfn>
+              <p>{{VirtualKeyboard/overlaysContent}}</p>
             </dt>
             <dd>
               <p>
-                When this attribute is set to `true`, a user agent MUST NOT resize its layout viewport or visual viewport.
+                The {{VirtualKeyboard/overlaysContent}} getter steps are to return [=this=]'s {{VirtualKeyboard/overlaysContent}}.
+              </p>
+              <p>
+                The {{VirtualKeyboard/overlaysContent}} setter steps are to set [=this=]'s {{VirtualKeyboard/overlaysContent}} to the given value.
               </p>
             </dd>
             <dt>
-              <dfn>boundingRect</dfn>
+              <p>{{VirtualKeyboard/boundingRect}}</p>
             </dt>
             <dd>
             <p>
-                The attribute reports the intersection of the VK with the layout viewport in client coordinates.
+                The attribute reports the intersection of the VK with the [=document=]'s <a href="https://drafts.csswg.org/css2/#viewport">viewport</a> in client coordinates. Call {{VirtualKeyboard/set the virtual keyboard bounding rect}}.
+                <p>
+                    <p><dfn>set the virtual keyboard bounding rect</dfn></p>
```

Comment:
"To <dfn>set the virtual keyboard bounding rect, with |osk| (a DOMRect representing the on-screen keyboard rectangle), and |lv| (a DOMRect representing the document's viewport rectangle) as inputs, run the following steps:"

Reply:
Done.